pkg/arvo/sys/arvo.hoon

```diff
@@ -191,7 +191,7 @@
   ?.  ?=({@ @ @ @ *} u.pux)  ~
   =+  :*  hyr=(slaw %tas i.u.pux)
           fal=(slaw %p i.t.u.pux)
-          dyc=(slaw %tas i.t.t.u.pux)
+          dyc=?~(i.t.t.u.pux (some %$) (slaw %tas i.t.t.u.pux))
           ved=(slay i.t.t.t.u.pux)
```

Comment:
Won't this line crash if `i.t.t.u.pux` is `~`?

Reply:
I'm not sure I understand. The `?~` is handling the `~` case explicitly, how would it crash?

Reply:
Oh, this is really working around a weirdness in path parsing, I guess, where if there's an empty path element, `+slaw` won't parse it to a `@tas`, even though `%$` is a valid `@tas` whose numeric value is zero.  So this is fine, just a bit counterintuitive.